src/main/java/com/artipie/rpm/Repomd.java

```diff
@@ -119,71 +119,95 @@ private Completable loadRepomd(final Path file) {
      * Perform the actual update.
      *
      * @param type The file type
-     * @param act The action to perform
-     * @param temp The temp
+     * @param act Action
+     * @param repomd The temp
```

Comment:
@g4s8 These paremeters could be better described, it is almost impossible to understand them without reading the entire method code